runtime/plaid/src/executor/mod.rs

```diff
@@ -409,24 +413,6 @@ fn process_message_with_module(
     els: Logger,
     performance_mode: Option<Sender<ModulePerformanceMetadata>>,
 ) -> Result<(), ExecutorError> {
-    // For every module that operates on that log type
-    // Mark this rule as currently being processed by locking the mutex
-    // This lock will be dropped at the end of the iteration so we don't
-    // need to handle unlocking it
-    let _lock = match module.concurrency_unsafe {
```

Comment:
Did you check all references to this were removed from the configuration as well?